### PR TITLE
Update Farming.ts

### PR DESCRIPTION
**Pull request by liczale**
Proving a point to someone who said Money should be changed to Pokedollars, despite having no experience myself I learned it.  -Liczale

### CODE REVIEW

File: src/scripts/farming/Farming.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ class Farming implements Feature {
 
         if (!suppressNotification) {
             Notifier.notify({
-                message: `You earned ${money} money from the harvest!`,
+                message: `You earned ${money} pokédollars from the harvest!`,
```
Comment (SpenserJ) [inline src/scripts/farming/Farming.ts:123]: It looks like our standard is to have this capitalized like you did in the other notification

**Review by SpenserJ — CHANGES_REQUESTED** [review]
Just one small change, but great work for your first time writing code and contributing to a project with git!

File: src/scripts/farming/Farming.ts (hunk 1 above)
Comment (liczale) [reply src/scripts/farming/Farming.ts]: Fixed it

**Review by SpenserJ — APPROVED** [review]
Looks great, and congratulations on your first PR! :+1:

**Pull request merged**